Documentation/project-docs/platform-names.md

```diff
@@ -8,19 +8,19 @@ This will also list some aliases. As you'll see the aliases aren't always correc
 
 **Also referred to as**: Desktop, full framework, in-box framework
 
-This refers to the .NET Framework that shipped in 2002. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
+This refers to the .NET Framework that first shipped in 2002 and is now up to version 4.6.1. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
```

Comment:
Do we really need the 4.6.1 version number here? It's bound to get outdated when new versions are added. Maybe just add something like `... first shipped in 2002 and has been updated on a regular basis since then.`